src/test/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/coordinator/ManagedIndexCoordinatorIT.kt

```diff
@@ -100,7 +70,8 @@ class ManagedIndexCoordinatorIT : IndexStateManagementRestTestCase() {
         }
     }
 
-    fun `test managed index metadata is cleaned up after removing policy_id`() {
+    fun `test managed index metadata is cleaned up after removing policy`() {
+        Assume.assumeTrue(false)
```

Comment:
If this is being used to ignore this test for now, I believe prepending `skip` to the test title will accomplish the same thing. So changing the test name to:

`skip test managed index metadata is cleaned up after removing policy`

Reply:
cleaned up, this test will not be skipped